test/solc/CommandLineParser.cpp

```diff
@@ -332,6 +332,11 @@ BOOST_AUTO_TEST_CASE(assembly_mode_options)
 		{
 			expectedOptions.optimizer.enabled = true;
 			expectedOptions.optimizer.yulSteps = "agf";
+			expectedOptions.optimizer.expectedExecutionsPerDeployment = 1000;
+		}
+		else
+		{
+			expectedOptions.optimizer.expectedExecutionsPerDeployment = 200;
 		}
```

Comment:
```suggestion
		else
			expectedOptions.optimizer.expectedExecutionsPerDeployment = 200;
```

Reply:
Maybe it's better to use `OptimiserSettings{}.expectedExecutionsPerDeployment` instead of 200 here? From `libsolidity/interface/OptimiserSettings.h`.